### PR TITLE
Fix FWS is not trimmed correctly

### PR DESCRIPTION
**Pull request by Weiling-Liao**

### CODE REVIEW

**Comment by mbaechler** [comment]
Thank you for you contribution @Weiling-Liao

File: main/src/test/java/org/apache/james/jdkim/tagvalue/TagValueTest.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public void testInvalidFWS() {
     @Test
     public void testInvalidFWSSyntax() {
         try {
-            new TagValue("p=test \r\n\r\n ");
+            new TagValue("p=test \r\n\r\n");
```
Comment (remk) [inline main/src/test/java/org/apache/james/jdkim/tagvalue/TagValueTest.java]: I'm not familiar with FWS. But does the assertion still make sense once the whitespace after the CRLF is removed ?

Comment (Weiling-Liao) [reply main/src/test/java/org/apache/james/jdkim/tagvalue/TagValueTest.java]: According to https://tools.ietf.org/html/rfc6376#section-2.8 , CRLF followed by at least one whitespace, so this new version without at least one whitespace after CRLF will be wrong FWS.

Comment (Weiling-Liao) [reply main/src/test/java/org/apache/james/jdkim/tagvalue/TagValueTest.java]: And this is a invalid FWS test

Comment (remk) [reply main/src/test/java/org/apache/james/jdkim/tagvalue/TagValueTest.java]: Thank you

**Pull request merged**